### PR TITLE
GP2-507 - Latest stats on Dashboard

### PR DESCRIPTION
**Pull request by marianasrv**
- Add Dashboard endpoint in the BE to get data from contentful.
- Create Info Card to show the stats.
- Add Latest Stats section to dashboard
- Update icons to be able to change color and size
- Add missing icons to storybook (gp2 icons)
- add article icon

### CODE REVIEW

File: packages/model/src/gp2/dashboard.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { ListResponse } from '../common';
+
+export type DashboardDataObject = {
+  sampleCount: number;
+  articleCount: number;
+  cohortCount: number;
+};
+
+export type ListDashboardDataObject = ListResponse<DashboardDataObject>;
```
Comment (jhonny-m) [inline packages/model/src/gp2/dashboard.ts:9]: we should not make this a `ListResponse`

File: packages/gp2-components/src/templates/DashboardPageBody.tsx [hunk 2]
```diff
@@ -40,21 +44,62 @@ const accodionItemStyles = css({
...
 });
 
+const latestStatsWrapperStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const latestStatsCardsStyles = css({
+  display: 'flex',
+  gap: '24px',
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/DashboardPageBody.tsx:54]: use `rem(x)`

File: packages/react-components/src/templates/TeamProfileHeader.tsx [hunk 3]
```diff
@@ -173,7 +173,9 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
         )}
         {labCount > 0 && (
           <div css={labCountStyles}>
-            <span css={iconStyles}>{labIcon} </span>
+            <span css={iconStyles}>
+              <LabIcon />{' '}
```
Comment (jhonny-m) [inline packages/react-components/src/templates/TeamProfileHeader.tsx:177]: remove the space

File: packages/react-components/src/icons/shared-research.tsx [hunk 4]
```diff
@@ -3,12 +3,16 @@ import { FC } from 'react';
...
 
-const LibraryIcon: FC<LibraryIconProps> = ({ color = '#4D646B' }) => (
+const LibraryIcon: FC<LibraryIconProps> = ({
+  color = '#4D646B',
+  size = 24,
+}) => (
   <svg
-    width="24px"
-    height="24px"
+    width={size}
```
Comment (jhonny-m) [inline packages/react-components/src/icons/shared-research.tsx:14]: is `24` and `24px` the same? just curious

Comment (marianasrv) [reply packages/react-components/src/icons/shared-research.tsx:14]: yes

File: apps/gp2-server/src/controllers/dashboard.controller.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  fetch: () => Promise<gp2.ListDashboardResponse>;
+}
+
+export default class Dashboard implements DashboardController {
+  constructor(private dashboardDataProvider: DashboardDataProvider) {}
+
+  async fetch(): Promise<gp2.ListDashboardResponse> {
+    const { total, items } = await this.dashboardDataProvider.fetch(null);
+
+    return {
```
Comment (jhonny-m) [inline apps/gp2-server/src/controllers/dashboard.controller.ts:14]: I was saying to do it here, if total and items are 0 then return it the default

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**